src/inmanta/db/versions/v202301100.py

```diff
@@ -0,0 +1,34 @@
+"""
+    Copyright 2022 Inmanta
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+    Contact: code@inmanta.com
+"""
+
+from asyncpg import Connection
+
+
+async def update(connection: Connection) -> None:
+    # grouped_by has as default value 'None' as it is part of the PRIMARY KEY and can therefore not be NULL.
+    schema = """
+    ALTER TABLE public.environmentmetricsgauge ADD COLUMN grouped_by VARCHAR DEFAULT 'None';
```

Comment:
I would use a default value here for which it is very unlikely that it would ever be the name of a group. Like `__None__` for example. `None` sound like something that could become a group at some point.